ethereum/contracts/pyth/Pyth.sol

```diff
@@ -236,17 +236,7 @@ abstract contract Pyth is PythGetters, PythSetters, AbstractPyth {
         return validTimePeriodSeconds();
     }
 
-    function isPyth() internal pure returns (bool) {
```

Comment:
i think you can't delete this until you upgrade the on-chain contracts (because the currently deployed version checks it)

Reply:
This one is very interesting but this check was not working as I wasn't using `this.isPyth` before.

Reply:
(I also commented in the `this.pythUpgradableMagic()` method why this is used)